app/models/school.rb

```diff
@@ -196,6 +197,13 @@ class ProcessDataError < StandardError; end
   enum funding_status: [:state_school, :private_school]
   enum region: [:north_east, :north_west, :yorkshire_and_the_humber, :east_midlands, :west_midlands, :east_of_england, :london, :south_east, :south_west]
 
+  # Defined as Postgres Enum. Mapping is from ruby values to database values
+  enum :data_sharing, {
+    public: 'public',
+    within_group: 'within_group',
```

Comment:
This might just be personal preference, but I wonder if 'group' rather than 'within_group' would be better as all the others are single words. Appreciate that would be a pain to change though